app/scripts/views/mixins/signin-mixin.js

```diff
@@ -24,13 +31,15 @@ define(function (require, exports, module) {
      * @return {object} promise
      */
     signIn: function (account, password) {
+      var self = this;
+      self.logEvent(`flow.${this.signInSubmitContext}.submit`);
```

Comment:
Is there a reason to use `signInSubmitContext` instead of [this.viewName](https://github.com/mozilla/fxa-content-server/blob/c3cc9d4a462e9c96cd702a5e0161065c4d8c0139/app/scripts/views/base.js#L138:L146)? Seems like the already existing name should suffice?